docs/source/workflow_inputs.rst

```diff
@@ -759,6 +759,7 @@ Each window or glass door area is entered as an ``/HPXML/Building/BuildingDetail
   ``ExteriorShading/WinterShadingCoefficient``  double             frac              0 - 1                No        1.00       Exterior winter shading coefficient (1=transparent, 0=opaque)
   ``InteriorShading/SummerShadingCoefficient``  double             frac              0 - 1                No        0.70 [#]_  Interior summer shading coefficient (1=transparent, 0=opaque)
   ``InteriorShading/WinterShadingCoefficient``  double             frac              0 - 1                No        0.85 [#]_  Interior winter shading coefficient (1=transparent, 0=opaque)
+  ``StormWindow``                               element                                                   No        <none>     Presence of storm window
```

Comment:
Since there is just a single element within ``StormWindow`` that we care about, I suggest not using a whole separate table just for that. So change this from ``StormWindow`` to ``StormWindow/GlassType`` and update accordingly. Same for skylights.